guides/software-versioning.md

```diff
@@ -217,7 +217,7 @@ general the following guidelines should be considered:
 * If a particular feature of a new Python version is desired by a package, but
   it conflicts with the above rules, where possible an effort should be made to
   use compatibility functions or external packages (new Python features are
-  often backported to older versions via external packages soon after the new
+  often back-ported to older versions via external packages soon after the new
```

Comment:
"backport" might actually be acceptable without the dash. It is a common jargon, even if Merriam Webster does not approve of it.

Reply:
Sounds good to me.